wcwidth/wcwidth.pyi

```diff
@@ -0,0 +1,21 @@
+from typing import Optional
+
+
+def wcwidth(wc: str, unicode_version: str = ...) -> int:
+    ...
+
+
+def wcswidth(pwcs: str, n: Optional[int] = None, unicode_version: str = ...):
+    ...
```

Comment:
These 2 functions accept [`typing.AnyStr`](https://docs.python.org/3/library/typing.html#typing.AnyStr) instead of `str`.

Reply:
Hmmmmmm... I was wrong, they don't accept `bytes` on Python 3, but accept both `str` and `unicode` on Python 2.

Reply:
Thanks, I'll fix this later

Reply:
So looks like there is no way to type this correctly?